.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+on:
+  pull_request
+
+jobs:
+  test:
+    runs-on: macos-12
+    name: Test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: mikehardy/buildcache-action@v2
```

Comment:
To accelerate this workflow using [build cache](https://github.com/mbitsnbites/buildcache).
https://github.com/mikehardy/buildcache-action